extension/chrome/elements/composer/composer-draft.ts

```diff
@@ -64,12 +65,13 @@ export class ComposerDraft extends ComposerComponent {
         return;
       }
       const parsedMsg = await Mime.decode(Buf.fromBase64UrlStr(draftGetRes.message.raw!));
-      const armored = PgpArmor.clip(parsedMsg.text || Xss.htmlSanitizeAndStripAllTags(parsedMsg.html || '', '\n') || '');
+      const encryptedMsg = parsedMsg.atts.find(att => att.treatAs() === 'encryptedMsg')?.getData().toUtfStr();
```

Comment:
Here we should re-use the same mechanism that we use when decrypting messages in `pgp_blocks`, for consistency.